fivox/helpers.h

```diff
@@ -140,6 +142,25 @@ inline void addCompartmentEvents(
     }
 }
 
+/**
+ * Get the set of GIDs from the URI parameters. Optimized for loading the
+ * circuit target by omitting target parsing.
+ *
+ * @param params the URI parameters to obtain BlueConfig and target name
+ * @return the loaded set of GIDs
+ */
+inline brion::GIDSet getGIDs( const URIHandler& params )
```

Comment:
You can pass the BlueConfig to this function. You already have one when the function is used.